src/index.js

```diff
@@ -366,7 +363,10 @@ export default class Carousel extends React.Component {
             );
 
         // prevents disabling click just because mouse moves a fraction of a pixel
-        if (length >= 10) this.clickDisabled = true;
+        if (length >= 10) {
+          if (this.clickDisabled === false) this.props.onDragStart(e);
+          this.clickDisabled = true;
```

Comment:
Don't love coupling these two together, but it works. Any better ideas?

Reply:
I can't think of anything right now, we can leave this in until I (or someone else with time) has time to figure out another solution